GITHUB101.md

```diff
@@ -0,0 +1,94 @@
+This document is aimed at getting started with github if you never used it.
+
+If you're a developer there's more info at [README.md](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/master/README.md).
+
+# Quick links
+
+- [Issues assigned to Gregg](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app//issues?q=assignee%3Agreggmiller+is%3Aopen)
+
+- [Issues tagged with design](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app//labels/design) need designer's attention
+
+- To install github mobile app head to [Android mobile link](market://details?id=com.github.android&hl=en) (alternative [Laptop link for android app](https://play.google.com/store/apps/details?id=com.github.android&hl=en) or [Apple placeholder]() store. You can login or sign up from there, for free.
+
+- Install the Western Health Covid19 app head at [Android placeholder]() or [Apple placeholder]() store. 
+- If you're on Android you can also download last minute versions from slack: go into the `#covid-19-app-builds` channel, find the last post from codemagic and download the `.apk` file. This is actually a good opportunity to hunt for the freshest bugs.
+
+- [Join GDG Melbourne's slack](http://bit.ly/join_gdgslack) with either work or personal email
+
+- [Join github](https://github.com/join?source=header-home) with either work or personal email
+
+
+# How can you help?
+
+Look into [issues assigned to Gregg](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues?q=assignee%3Agreggmiller+is%3Aopen), then review them and see what it is the developers may be asking for. Most often this will be a further copy (written information) and content or a breakdown of an algorithm/ checklist. 
+
+If you think you can help you can add a comment with either a simple reply or follow up question. Developers are a friendly bunch and won't be critical if you don't provide what they are looking for. You'll need to be logged in in order to make comments.
+  
+You can always ask for clarification by contacting Gregg directly on the general channel on Slack GDG Melbourne.  A good way to get a feel for the process and how issues are resolved might be to look at previous issues with Gregg's name assigned and the type of answer he has given. Generally, if it doesn't answer the developer then they will have responded.
```

Comment:
Actually there is a private `covid-19-app` channel - Might want to mention it

Reply:
adding "ask gregg for it", but may also "ask gregg or luke for it" since I don't know how you guys work it out.

> There's also a private slack channel `#covid-19-app`. If you're going to get commited to the project it's worth checking with Gregg how to get access to it.